doc/source/examples/analysis/polymers_and_membranes/README.rst

```diff
@@ -0,0 +1,14 @@
+.. -*- coding: utf-8 -*-
+
+==========================
+Polymers and membranes
+==========================
+
+MDAnalysis has several analyses specifically for polymers, membranes, and membrane proteins.
+
+.. toctree::
+   :maxdepth: 1
+
+   /examples/analysis/polymers_and_membranes/polymer
+   /examples/analysis/polymers_and_membranes/hole
```

Comment:
In the deprecation note, 
```
This module is now deprecated. Please use MDAnalysis.analysis.hole2 instead.
```
the module name  `MDAnalysis.analysis.hole2` is not a hyperlink.